lib/parser.ex

```diff
@@ -104,7 +104,7 @@ defmodule Prismic.Parser do
         %{value: %{document: %{slug: slug, type: doc_type, data: doc_data} = doc} = value} = _json
       ) do
     fragments =
-      if raw_frags = doc_data[doc_type] do
+      if raw_frags = doc_data[String.to_atom(doc_type)] do
```

Comment:
Was this a bugged?

Reply:
Yes. When we translate JSON back to an elixir struct, we convert string keys to atoms, but we don't convert values. So `doc_type` (a value) is a string, but the keys in `doc_data` are all atoms.